package.json

```diff
@@ -20,7 +20,8 @@
     "test": "karma start --single-run"
   },
   "peerDependencies": {
-    "react": ">=15 || ^0.14.7"
+    "prop-types": "^15.5.8",
+    "react": ">=15"
```

Comment:
Since if someone uses `react 14.x` he probably don't install the new `prop-types` package which is needed with the new React.
I preferred not to do something like:

```js
let PropTypes;
try {
  PropTypes = require("prop-types");
} catch(e) {
  PropTypes = require("react").PropTypes;
}

```